### DISCUSSION TITLE
how control 4 independent PWM Channels?

### DISCUSSION THREAD

**Discussion by MadJoe1974** [arendst/Tasmota#12372] [Q&A] [answered]
i have an ESP8285 with the following configuration:

gpio5			pwm1
gpio12			pwm2
gpio13			pwm3
gpio15			pwm4

its connectet to IOBroker with sonoff adapter

now my problems are:

1st i want to have only 4 simple sliders in web ui to dimm each channel, my webUI always looks like an rgb controller and is unusable for me

and 

2nd in iobroker i also want simple 4 fields to control 

sitting around 4 days and all i got is a behavier like an rgb controller...

the background is that i have 4 different Aquariums and want to dimm each of it

i have only basic programming skills so help is needed ..

best regards Joe

**Comment by sfromis** [comment, marked as answer]
Use [SetOption68](https://tasmota.github.io/docs/Commands/#setoption68) to separate the PWM channels

**Reply by MadJoe1974** [reply]
thanks, this helps a lot in webUI

is it possible to eliminate these 4 Power Buttons?

**Reply by sfromis** [reply]
Not with a straightforward setting. But you can use the `WebButton` command to label them. Anyway, don't fret too much over the web UI, which is not meant as a device dashboard, but merely for a bit of admin/testing.
